docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>"how a model parameter varies..." I think it would be slightly better to say "how a parameter of the (conditional) response distribution varies...". It's not incorrect to say "model parameter", but it can make people think of other model parameters such as regression coefficients. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>This is a good point. Model parameter is ambiguous. </p><p> </p>